src/lithium/reducer.py

```diff
@@ -1166,6 +1199,38 @@ def tryArgumentsAsGlobals(roundNum, testcase, interesting, tempFilename):  # pyl
         return numMovedArguments, testcase
 
 
+class CollapseEmptyBraces(Minimize):
```

Comment:
A test should be possible for this class, shouldn't it?

Reply:
Added via f98b5d655e2990052be35d88486c319195b24270